src/Sql/Sql/ManagedInstance/Cmdlet/SetAzureSqlManagedInstance.cs

```diff
@@ -130,6 +131,25 @@ public class SetAzureSqlManagedInstance : ManagedInstanceCmdletBase
             HelpMessage = "Determines how much VCore to associate with instance")]
         public int? VCore { get; set; }
 
+        /// <summary>
+        /// Gets or sets whether or not the public data endpoint is enabled.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "Whether or not the public data endpoint is enabled for the instance.")]
+        [ValidateNotNullOrEmpty]
+        [PSArgumentCompleter(Constants.True, Constants.False)]
```

Comment:
This should only be used for string parameters where the possible values are not clear - boolean parameter will only ever accept $true or $false so we don't need this completion.

Reply:
Autocomplete removed.